src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -65,6 +65,7 @@ public partial class DynamoView : Window, IDisposable
         private int tabSlidingWindowStart, tabSlidingWindowEnd;
         private GalleryView galleryView;
         private readonly LoginService loginService;
+        private readonly Stopwatch sw = new Stopwatch();
```

Comment:
It'snot a good idea to name local variables with something so indescriptive. Add a comment that explains what this stopwatch does.